src/connection-server.ts

```diff
@@ -1,27 +1,27 @@
-import { Sequence, Execution, Operation, fork, timeout } from 'effection';
+import { Context, Operation, fork, send, timeout } from 'effection';
 import { on } from '@effection/events';
 
-import { createSocketServer, Connection, Message, send } from './ws';
+import { createSocketServer, Connection, Message, sendData } from './ws';
```

Comment:
with `send` being an operation now, the web socket `send` had to be renamed to `sendData`